ds_judgements_public_ui/sass/includes/_gds_page.scss

```diff
@@ -0,0 +1,34 @@
+.govuk-fieldset__legend {
```

Comment:
Actually one minor organizational thing - i think all of the rules in this file could go in _gds_overrides.scss, and this file be deleted. Would that be ok? it seems simpler!

Reply:
.... in fact, it appears all the rules are already there (and this one isn't being included anywhere) - i think you can just delete it!

Reply:
deleted it reappeared after conflicts were being sorted @timcowlishaw